src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -344,6 +344,10 @@ export class BuildGraphTableComponent implements OnChanges {
     return node.coherent.withProduct;
   }
 
+  public hasIncoherencies(node: BuildData) {
+    return (node.build.incoherencies !== undefined && node.build.incoherencies.length > 0)
```

Comment:
this can just be `node.build.incoherencies && node.build.incoherencies.length` wrap it in `!!(...)` if you want to force it to a boolean, but thats not needed.